### PR TITLE
Add const to locals

### PR DESCRIPTION
**Pull request by SunnyWar**
Added const to local variables that can be const. This will identify non-const variables to insure that they are that way intentionally.

### CODE REVIEW

File: src/syzygy/syzygy.cc [hunk 1]
```diff
@@ -421,7 +421,7 @@ void init_indices() {
   }
   // TODO: choose a good seed.
   std::mt19937 gen(123523465);
-  std::uniform_int_distribution<Key> dist(std::numeric_limits<Key>::lowest(),
+  const std::uniform_int_distribution<Key> dist(std::numeric_limits<Key>::lowest(),
```
Comment (borg323) [inline src/syzygy/syzygy.cc]: g++/clang on linux complain about this (and the other `const` on distributions in `random.cc`)

Comment (SunnyWar) [reply src/syzygy/syzygy.cc]: I'll look into it.

Comment (SunnyWar) [reply src/syzygy/syzygy.cc]: I tried a variety of way to repro this in windows, including using the intel compiler, and attempting to set up a Linux sub system, but to no avail. So, I'm simply fixing the issues as demonstrated in the build log.

Comment (borg323) [reply src/syzygy/syzygy.cc]: You missed the one in `syzygy.cc`.

**Comment by mooskagh** [comment]
This looks good, but I'll merge it a bit later (after RC season) as such a widespread change may make cherrypicking harder.

**Comment by SunnyWar** [comment]
sure thing

**Comment by mooskagh** [comment]
@SunnyWar 
Note that that the code does not meet our style guide requirements (e.g. there are lines longer than 80 characters), but  I think it's better to merge it, otherwise it's always in "needs merging" state.

Please reformat code with `clang-format` and submit fix.

**Pull request merged**